Products/ZenModel/UserInterfaceSettings.py

```diff
@@ -34,6 +34,7 @@ class UserInterfaceSettings(ZenModelRM):
         {'id': 'showEventSeverityIcons', 'type': 'boolean', 'mode': 'w'},
         {'id': 'enableTreeFilters', 'type': 'boolean', 'mode': 'w'},
         {'id': 'showPageStatistics', 'type' : 'boolean', 'mode':'w'},
+        {'id': 'renderNetworksFromCatalog', 'type' : 'boolean', 'mode':'w'},
```

Comment:
Unfortunately, I have not found the way to get search through catalog works fine -- it takes a lot of time. This option gives user a choice: whether have Network page available in case of extra-large networks with limited search, or do not change anything and have fully-functional search.